client/src/entry/analysis/modules/Analysis.vue

```diff
@@ -9,10 +9,11 @@ import CenterFrame from "./CenterFrame.vue";
 import { useRoute, useRouter } from "vue-router/composables";
 import { computed, ref, onMounted, onUnmounted } from "vue";
 import { useUserStore } from "@/stores/userStore";
+import { storeToRefs } from "pinia";
 
 const route = useRoute();
 const router = useRouter();
-const userStore = useUserStore();
+const { toggledActivityBar } = storeToRefs(useUserStore());
```

Comment:
Can you also rename this to `showActivityBar` ? `toggled` doesn't technically say if it should be visible or not.